code/game/machinery/syndicatebomb.dm

```diff
@@ -1,43 +1,60 @@
 #define BUTTON_COOLDOWN 60 // cant delay the bomb forever
 #define BUTTON_DELAY	50 //five seconds
+#define PLASTEEL_REPAIR_AMOUNT 2
 
 /obj/machinery/syndicatebomb
-	icon = 'icons/obj/assemblies.dmi'
 	name = "syndicate bomb"
-	icon_state = "syndicate-bomb"
 	desc = "A large and menacing device. Can be bolted down with a wrench."
+	icon = 'icons/obj/assemblies.dmi'
+	icon_state = "syndicate-bomb"
 
 	anchored = FALSE
-	density = FALSE
 	layer = BELOW_MOB_LAYER //so people can't hide it and it's REALLY OBVIOUS
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	processing_flags = START_PROCESSING_MANUALLY
 	subsystem_type = /datum/controller/subsystem/processing/fastprocess
 
 	interaction_flags_machine = INTERACT_MACHINE_WIRES_IF_OPEN | INTERACT_MACHINE_OFFLINE
 
-	var/minimum_timer = 90
-	var/timer_set = 90
+	var/minimum_timer = 30
+	var/timer_set = 30
```

Comment:
30 seconds seems very short, even with 90 seconds there isn't enough time to
1) Find a bomb
2) Get tools to defuse the bomb
3) Attempt to defuse it
I fear this will make it so that there isn't even enough time to evacuate the area

Reply:
What do you think would be a good number for it, then?

Reply:
I guess 60 at least